### PR TITLE
Fixed language codes in a batch of import x28 jobs into a local postgresql

### PR DESCRIPTION
**Pull request by wwebec**

### CODE REVIEW

File: master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	static final String LANGUAGE_CODE_COLUMN_FORMAT = "SK%d_SPRACHE_CODE";
+	static final Integer JOB_RECORD_BEGIN_COLUMN = 0;
+	static final Integer JOB_RECORD_END_COLUMN = 5;
+
+	List<JobLanguageRecord> mapRecords(ResultSet rs) {
+		return mapRecords(rs, JOB_RECORD_BEGIN_COLUMN, JOB_RECORD_END_COLUMN);
+	}
+
+	List<JobLanguageRecord> mapRecords(ResultSet rs, int start, int end) {
+		return range(start, end)
```
Comment (birom) [inline master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java]: According to the select string in the LegacyToMasterConfiguration we have SK1_SPRACHE_CODE .... SK5_SPRACHE_CODE. But we generate here SK0_SPRACHE_CODE... SK4_SPRACHE_CODE.

File: master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+			return null;
+		}
+	}
+
+	private JobLanguageRecord recordWithLanguageCode(String iso6391) {
+		JobLanguageRecord record = new JobLanguageRecord();
+		record.setLanguageId(Iso6391LanguageCode.toLanguageCode(iso6391));
+		return record;
+	}
+}
```
Comment (birom) [inline master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java]: Please add new line at the end.

File: master/legacy/src/test/java/ch/alv/batches/legacy/to/master/JobRecordMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+		assertThat(record.getLanguageId(), nullValue());
+	}
+
+	@Test
+	public void shouldMapRecordsReturnOneRecordIfOneLanguageCodeCanBeRetrieved() {
+		List<JobLanguageRecord> records = mapper.mapRecords(rs);
+
+		assertThat(records, hasSize(1));
+	}
+}
```
Comment (birom) [inline master/legacy/src/test/java/ch/alv/batches/legacy/to/master/JobRecordMapperTest.java]: Please add new line at the end

File: master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java (hunk 1 above)
Comment (wwebec) [reply master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java]: Right, corrected to range from 1 to 5.

File: master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import ch.alv.batches.legacy.to.master.jooq.tables.records.JobLanguageRecord;
+
+class JobRecordMapper {
```
[the remark this answers is not in the document]
Comment (wwebec) [reply master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java]: Right, ranamed JobRecordMapper to JobLangaugeRecordMapper and corresponding test

File: master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java (hunk 2 above)
Comment (wwebec) [reply master/legacy/src/main/java/ch/alv/batches/legacy/to/master/JobRecordMapper.java]: added

File: master/legacy/src/test/java/ch/alv/batches/legacy/to/master/JobRecordMapperTest.java (hunk 3 above)
Comment (wwebec) [reply master/legacy/src/test/java/ch/alv/batches/legacy/to/master/JobRecordMapperTest.java]: added

File: jobdesk/master/src/test/java/ch/alv/batches/master/to/jobdesk/MasterToJobdeskIntegrationTest.java [hunk 5]
```diff
@@ -1,116 +1,126 @@
...
-        processBuilder.redirectInput(file);
-
-        Process psqlProcess = processBuilder.start();
-        boolean executed = psqlProcess.waitFor(30, TimeUnit.SECONDS);
-        assertEquals(true, executed);
-        int result = psqlProcess.exitValue();
-        assertEquals(0, result);
-        // TODO assert totals in DB: location=4168, job=24443, job_loc=13591
-    }
+	@Resource
```
Comment (birom) [inline jobdesk/master/src/test/java/ch/alv/batches/master/to/jobdesk/MasterToJobdeskIntegrationTest.java]: Hi @wwebec, 

would it be possible to keep the indentation. Earlier it was done by spaces and now with tabs. 

Thank you!

**Comment by fabiangotzen** [comment]
I'm merging this pull request so we can keep track of the changes and don't have to solely rely on the database view. However, since its not clear yet if the daily cronjobs work again as expected I won't deploy the version to the prod server for now so we can focus on stabilizing the application.

**Pull request merged**